<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>939</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRUD Men√∫üìãüë®‚Äçüç≥üçΩÔ∏è</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>701</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>48</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>üìãCRUD MEN√öüë®‚Äçüç≥üçΩÔ∏è</string>
    </property>
   </widget>
   <widget class="QLabel" name="saludo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>¬°BIENVENIDO!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agregar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>‚ú®AGREGAR </string>
    </property>
   </widget>
   <widget class="QPushButton" name="buscar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>üîç BUSCAR </string>
    </property>
   </widget>
   <widget class="QPushButton" name="actualizar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>üõ†Ô∏èACTUALIZAR </string>
    </property>
   </widget>
   <widget class="QPushButton" name="eliminar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>üóëÔ∏è ELIMINAR </string>
    </property>
   </widget>
   <widget class="QPushButton" name="mostrar">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>410</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>üëÄMOSTRAR MEN√öS</string>
    </property>
   </widget>
   <widget class="QLabel" name="nombre_menu">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre Men√∫:</string>
    </property>
   </widget>
   <widget class="QLabel" name="categoria">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Categor√≠a:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="digitar_nombre">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="id_menu">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ID Men√∫ :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="digitar_id">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="escribir_descripcion">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="descripcion">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Descripci√≥n:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="escribir_precio">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="precio">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Precio:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="categoria_item">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Desayuno</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Almuerzo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Merienda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Postres</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C√≥cteles</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bebidas</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tabla_registros">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>911</width>
      <height>391</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NOMBRE</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CATEGOR√çA</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DESCRIPCI√ìN</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PRECIO</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
